sirepo/auth/__init__.py

```diff
@@ -33,9 +33,10 @@
 METHOD_GUEST = 'guest'
 
 ROLE_ADM = 'adm'
-ROLE_PREMIUM = 'premium'
+ROLE_PAYMENT_PLAN_ENTERPRISE = 'enterprise'
+ROLE_PAYMENT_PLAN_PREMIUM = 'premium'
 
-PAID_USER_ROLES = (ROLE_PREMIUM,)
+PAID_USER_ROLES = (ROLE_PAYMENT_PLAN_PREMIUM,)
```

Comment:
```py
PAID_USER_ROLES = (ROLE_PAYMENT_PLAN_PREMIUM, ROLE_PAYMENT_PLAN_ENTERPRISE)
```

Reply:
https://github.com/radiasoft/sirepo/pull/2711/commits/05bde41c8e65d6cbc52101463c4a1a6418c2356b

Reply:
Disregard my last comment.

https://github.com/radiasoft/sirepo/pull/2711/commits/a2b0f74dc4310d3c3f3502e3d89184a0d8439782 fixes it